device/mellanox/x86_64-mlnx_msn2700-r0/platform_reboot

```diff
@@ -4,6 +4,7 @@ declare -r EXIT_SUCCESS="0"
 declare -r EXIT_ERROR="1"
 
 declare -r FW_UPGRADE_SCRIPT="/usr/bin/mlnx-fw-upgrade.sh"
+declare -r SYSFS_PWR_CYCLE="/sys/devices/platform/mlxplat/mlxreg-io/hwmon/hwmon1/pwr_cycle"
```

Comment:
Just checking, will `pwr_cycle` always be located under `hwmon1`, or is there a chance for it to be `hwmon2`, etc.?

Reply:
can you address this question?

Reply:
You're right, it can be different sometimes.